brownie/network/account.py

```diff
@@ -944,26 +944,24 @@ def _apply_fee_to_tx(
 ):
     tx = tx.copy()
 
-    if gas_price is not None and (max_fee or priority_fee):
-        raise ValueError("gas_price and (max_fee, priority_fee) are mutually exclusive")
+    if gas_price is not None:
+        if max_fee or priority_fee:
+            raise ValueError("gas_price and (max_fee, priority_fee) are mutually exclusive")
+        tx["gasPrice"] = web3.toHex(gas_price)
+        return tx
 
-    elif max_fee is not None and priority_fee is not None:
-        if priority_fee > max_fee:
-            raise InvalidTransaction("priority_fee must not exceed max_fee")
+    if priority_fee is None:
+        raise InvalidTransaction("priority_fee must be defined")
```

Comment:
Check if these invalid transaction checks can be ommited and caught downstream in web3.py